swift-extensions/UISliderExtensions.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+import TRIKOT_FRAMEWORK_NAME
+
+public class Slider : UISlider {
```

Comment:
I dont think its a good Idea to create views here. Why forcing the user to use a Subclass of UISlider ? Can we only add binding like we did for All the other components on IOS ?

Reply:
^this

Reply:
it's mainly because of the `oldValue` stored property that is needed to fix a problem where the value was updated on every pixel moved on the slider (calling `setSelectedValue` and triggering the impactFeedback hundreds of times).

Reply:
You can use Objc_setAsociatedValue for this